### ISSUE TITLE
Relative Paths

### ISSUE THREAD

**Issue by aylen384** [Anishka0107/Wunner#1]
It looks as if Wunner has problems with relative paths on websites.

**Comment by Anishka0107** [comment]
Yes it is incapable of handling relative URLs on websites, thanks for opening the issue!
Infact it is a part of the Todo list : https://github.com/Anishka0107/Wunner/blob/master/README.md#todo-checklist

If anyone would like to work on this, contributions are most welcome.
